### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: grails-app/services/de/laser/SurveyService.groovy [hunk 1]
```diff
@@ -1715,6 +1715,10 @@ class SurveyService {
                      acceptStatus: RDStore.IE_ACCEPT_STATUS_FIXED])[0]
 
             if(countIes > 0){
+                println(tipp.id)
```
Comment (croqueGrec09) [inline grails-app/services/de/laser/SurveyService.groovy:1718]: needed or debug?

Comment (Moetez16) [reply grails-app/services/de/laser/SurveyService.groovy]: PR updated!

File: grails-app/services/de/laser/ctrl/SubscriptionControllerService.groovy [hunk 2]
```diff
@@ -1362,7 +1362,7 @@ class SubscriptionControllerService {
             if(result.surveyConfig.pickAndChoosePerpetualAccess) {
                 subscriptions = linksGenerationService.getSuccessionChain(newSub, 'sourceSubscription')
                 subscriptions << newSub
-                result.subscriptionIDs = surveyService.subscriptionsOfOrg(result.institution)
+                result.subscriptionIDs = surveyService.subscriptionsOfOrg(result.subscriber)
```
Comment (croqueGrec09) [inline grails-app/services/de/laser/ctrl/SubscriptionControllerService.groovy:1365]: result.subscriber?

**Review by croqueGrec09 — COMMENTED** [review]
remarks

**Pull request merged**